lib/providers/providers.dart

```diff
@@ -26,66 +26,62 @@ import 'package:seeds/providers/services/permission_service.dart';
 
 // Connection => Settings => Auth => Http => Members
 final providers = [
-  Provider(
-    create: (_) => NavigationService(),
-  ),
-  ChangeNotifierProvider(
-    create: (_) => ConnectionNotifier()..init(),
-  ),
-  ChangeNotifierProxyProvider<ConnectionNotifier, SettingsNotifier>(
-    create: (_) => SettingsNotifier()..init(),
-    update: (_, connection, settings) => settings!
-      ..update(
-        nodeEndpoint: connection.currentEndpoint,
-      ),
-  ),
-  ChangeNotifierProxyProvider<SettingsNotifier, AuthNotifier>(
-      create: (_) => AuthNotifier(),
-      update: (_, settings, auth) {
-        if (settings.isInitialized) {
-          return auth!
-            ..update(
-              accountName: settings.accountName,
-              privateKey: settings.privateKey,
-              passcode: settings.passcode,
-              passcodeActive: settings.passcodeActive,
-              inRecoveryMode: settings.inRecoveryMode,
-            );
-        } else {
-          return auth!;
-        }
-      }),
-  ProxyProvider<SettingsNotifier, LinksService>(
-    create: (_) => LinksService(),
-    update: (_, settings, links) => links!
-      ..update(
-        accountName: settings.accountName,
-        enableMockLink: false,
-      ),
-  ),
-  ProxyProvider<SettingsNotifier, HttpService>(
-    create: (_) => HttpService(),
-    update: (_, settings, http) => http!
-      ..update(
-        accountName: settings.accountName,
-        nodeEndpoint: settings.nodeEndpoint,
-        enableMockResponse: false,
-      ),
-  ),
-  ProxyProvider<SettingsNotifier, EosService>(
-    create: (context) => EosService(),
-    update: (context, settings, eos) => eos!
-      ..update(
-        userPrivateKey: settings.privateKey,
-        userAccountName: settings.accountName,
-        nodeEndpoint: settings.nodeEndpoint,
-        enableMockTransactions: false,
-      ),
-  ),
-  ChangeNotifierProxyProvider<HttpService, MembersNotifier>(
-    create: (context) => MembersNotifier(),
-    update: (context, http, members) => members!..update(http: http),
-  ),
+  Provider(create: (_) => NavigationService()),
+  ChangeNotifierProvider(create: (_) => ConnectionNotifier()..init()),
```

Comment:
These 2 are the only ones that are in v2.
ConnectionNotifier is only used in app widget and the Navigator class could be left like this or we can use the RepositoryProvider widget from the flutter_bloc lib.